zulipterminal/ui_tools/views.py

```diff
@@ -883,8 +891,7 @@ def __init__(self, controller: Any, popup_title: str) -> None:
         self.width = popup_width
         help_menu_content = self.make_table_with_categories(help_menu_content,
                                                             column_widths)
-        self.height = len(help_menu_content)
-
+        self.height = self.get_popup_height(help_menu_content, popup_width)
```

Comment:
As per the other comment re `help_menu_content` being re-used, using a different name would emphasize why the order is important here.